tsconfig.json

```diff
@@ -1,10 +1,8 @@
 {
   "compilerOptions": {
```

Comment:
Why are these lines removed? Could you explain what the reason is behind it without me having to figure it out on my own? Thank you :)